docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -36,50 +62,92 @@ Check for the below `INFO` log to confirm if the server started successfully,
+. *?auto-create*: the query parameter that indicates the cache manager on whether or not to create the clustered storage with the given identifier if one is not found at the server.
+  This is an optional parameter.
+
+Here is a code sample that shows how to configure a cache manager with clustering service.
 
 [source,java,indent=0]
 ----
 include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java[tag=clusteredCacheManagerExample]
 ----
 
 <1> Returns the `org.ehcache.config.builders.CacheManagerBuilder` instance;
-<2> Use the `ClusteringServiceConfigurationBuilder`{empty}'s static method `.cluster(URI)` for connecting the cache manager to the clustering storage at the
+<2> Use the `ClusteringServiceConfigurationBuilder`{empty}'s static method `.cluster(URI)` for connecting the cache manager to the clustered storage at the
     URI specified that returns the clustering service configuration builder instance.
-    Sample URI provided in the example is pointing to the clustered storage with clustered storage identifier *my-application* on the Terracotta server (Assuming the server is running on localhost and port *9510*); the query-param `auto-create`
-    creates the clustered storage in the server if it doesn't already exist.
+    Sample URI provided in the example is pointing to the clustered storage named `my-application` on the Terracotta server (Assuming the server is running on localhost and port *9510*).
+    The query-param `auto-create` would make sure that the cache manager creates the clustered storage instance named `my-application` in the server if it doesn't already exist.
 <3> Returns a fully initialized cache manager that can be used to create clustered caches.
 <4> Close the cache manager.
 
 == Cache manager configuration and usage of server side resources
 
-After starting the Terracotta server (Refer section: <<Starting Terracotta server>>), Cache manager with clustering capabilities can be configured with server side resource(s) and in turn can be used by cache(s) :
+After starting the Terracotta server (Refer section: <<start-server, Starting Terracotta server>>), cache managers with clustering capabilities can be configured with server side resource(s) which in turn can be used by cache(s).
+The following sample; unlike the previous one; uses a broader clustering service configuration that lets you configure more things at the cache manager level.
+Before we jump to configuration samples, we'll define certain terms that are used in the remainder of this document.
+
+Server off-heap resource::
+Server off-heap resources are storage resources defined at the server.
+`primary-server-resource` and `secondary-server-resource` are two named server off-heap resources defined in the server configuration from previous section.
+Clustered caches can have fixed amount of storage allocated directly from these resources.
+
+Resource pool::
```

Comment:
You are missing a similar section on cache fixed pool since the picture below contains some.